File: Frontend/UsersControlWebApp/wwwroot/js/Users/UserReadViewModel.js
class UserReadViewModel {
    constructor(data) {
        data = data || {};
        const self = this;
        self.id = ko.observable(data.id);
        self.name = ko.observable(data.name);
        self.lastName = ko.observable(data.lastName);
        self.email = ko.observable(data.email);
        self.birthday = ko.observable(data.birthday);
        self.telephoneNumber = ko.observable(data.telephoneNumber);
        self.sendNews = ko.observable(data.sendNews);
        self.countryNameAndCode = ko.observable(data.countryNameAndCode);
    }
}

File: Frontend/UsersControlWebApp/ViewModels/Users/UserReadViewModel.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace UsersControlWebApp.ViewModels.Users
{
    public class UserReadViewModel
    {

        public string Name { get; set; }

        [Display(Name = "Last Name")]
        public string LastName { get; set; }

        public string Email { get; set; }

        public string Birthday { get; set; }

        [Display(Name = "Telephone Number")]
        public int? TelephoneNumber { get; set; }

        [Display(Name = "Receive information?")]
        public bool SendNews { get; set; }

        [Display(Name = "Residence Country")]
        public string CountryNameAndCode { get; set; }

    }
}